src/functions-reference/functions_index.qmd

```diff
@@ -4198,6 +4198,18 @@ pagetitle: Alphabetical Index
  - <div class='index-container'>[distribution statement](positive_lower-bounded_distributions.qmd#index-entry-a8609b3c51ac8487f51a5e8a445e626015dad49f) <span class='detail'>(positive_lower-bounded_distributions.html)</span></div>
 
 
+<a id='wiener_lccdf_defective' href='#wiener_lccdf_defective' class='anchored unlink'>**wiener_lccdf_defective**:</a>
```

Comment:
Why are there only `_defective` versions in the doc?  Shouldn't the index have been expanded with all the regular functions, too or is that somehow automatic?